proto/cosmos/upgrade/v1beta1/upgrade.proto

```diff
@@ -22,10 +22,10 @@ message Plan {
   // reached and the software will exit.
   string name = 1;
 
-  // Time based upgrades have been deprecated. Time based upgrade logic
+  // Deprecated: Time based upgrades have been deprecated. Time based upgrade logic
   // has been removed from the SDK.
-  reserved 2;
-  reserved "time";
+  // If this field is not empty, an error will be thrown.
+  google.protobuf.Timestamp time = 2 [deprecated = true, (gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
in fact, this field is problematic. User simply can't use it.   We need to be clear in the changelog about it - because in other circumstance this should require a proto version update in my opinion.

Reply:
Yeah I agree, I updated the existing changelog entry, do you think it's enough? A mention in the release notes would be useful too.